sampledata.js

```diff
@@ -0,0 +1,48 @@
+const sampleFeed = [
+  {
+    transactionId: 123,
```

Comment:
We should make the transactionId's and userId's smaller numbers, because these numbers auto-increment (i.e. they won't be in the insert queries) and they start from 1. But this can be done in a future commit.

Reply:
Let's discuss in person. I think having larger IDs for sample data is valid since this will need to be handled in the front-end. In the end, we will drop this entire file to pull from database.

Reply:
Discussed the issue. This is isolated to front-end testing only and won't be passed to the DB. Therefore the values don't matter, just their data type.